api/src/integration/java/org/pmiops/workbench/ApplicationTest.java

```diff
@@ -19,6 +22,8 @@
 public class ApplicationTest {
 
   @Autowired private ApplicationContext context;
+  @MockBean private DataSource dataSource;
+  @MockBean private CdrDataSource cdrDataSource;
```

Comment:
New database configuration is less forgiving, so we have to mock these instead of providing bogus parameters.